inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv16.cpp

```diff
@@ -34,7 +34,9 @@ ParamsKey EltwiseKernel_b_fs_yx_fsv16::GetSupportedKey() const {
 static inline size_t GetBlockSize(const eltwise_params& params) {
     // Set blocksize 1 when broadcasting X dim
     for (size_t i = 0; i < params.inputs.size(); i++) {
-        if (params.inputs[i].X().v == 1 && params.inputs[i].LogicalSize() != 1) {
+        if ((params.inputs[i].X().v == 1) &&
+            (params.inputs[i].LogicalSize() != 1) &&
+            (params.inputs[i].LogicalSize() != params.output.Feature().v || params.inputs[i].Feature().v != params.output.Feature().v)) {
```

Comment:
The condition `(params.inputs[i].LogicalSize() == output.Feature().v && params.inputs[i].Feature().v == output.Feature().v) || (params.inputs[i].LogicalSize() == 1)` repeats 3 times in this patch. What about making a macro for it? What about `IS_BROADCASTING_POSSIBLE_FOR_INPUT`?

Reply:
Agree, updated it.

Reply:
maybe some inline function instead of macro?

Reply:
Okey, replaces inline func instead of macro